WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -228,23 +228,23 @@
           <TextBlock Grid.ColumnSpan="2" Grid.Column="0" Grid.Row="0" Text="Coinjoin Fees" Classes="title1" HorizontalAlignment="Center" VerticalAlignment="Center" />
 
           <StackPanel Grid.Column="0" Grid.Row="1">
-            <TextBlock Name="CoinjoinFeeTextBlock" Text="Service Fee" Classes="title2" />
-            <TextBlock Text="You will pay a flat fee of 0.3% for every coin you receive." Classes="text" />
+            <TextBlock Name="CoinjoinFeeTextBlock" Text="Coordination Fee" Classes="title2" />
+            <TextBlock Text="You will pay a flat fee of 0.3% for every new coin you coinjoin." Classes="text" />
```

Comment:
I think using the term "flat fee" is wrong here.

```suggestion
            <TextBlock Text="You will pay a fee of 0.3% only for new coins you coinjoin." Classes="text" />
```

Reply:
done